server/email/config.js

```diff
@@ -1,8 +1,9 @@
 Meteor.startup(function() {
 
```

Comment:
Yes, all development settings should be added on a different setting file. I believe the configuration for mailgun wasn't changed after adding the Accounts Meteor package.